### PR TITLE
fixes #7

### LINKED ISSUE #7: Doesn't build on windows
Original: https://github.com/jquery/globalize/issues/516

> Unless I am missing something, I don't think globalize-webpack-plugin would build on windows... I am thinking regular expressions might be the problem:

    
>
    GlobalizeCompilerHelper.prototype.getModuleFilepath = function(request) {
      return path.join(this.tmpdir, request.replace(/.*!/, "").replace(/[/?" ]/g, "-"));
    };

>**ERROR in ./app/index.js**
Module parse failed: C:\app-npm-webpack\app\index.js ENOENT, no such file or directory 'C:\app-npm-webpack\.tmp-globalize-webpack\C:\app-npm-webpack\app\index.js'
You may need an appropriate loader to handle this file type.
Error: ENOENT, no such file or directory 'C:\app-npm-webpack\.tmp-globalize-webpack\C:\app-npm-webpack\app\index.js'

Jörn has added this comment:
> Classic Windows path issue. Should be easy enough to fix. While at it, set up CI on Windows? http://www.appveyor.com/ makes that reasonably easy.

### PR DESCRIPTION
**Pull request by aruberto**

### CODE REVIEW

**Comment by rxaviers** [comment]
@alunny, this change is supposed to fix the plugin for Windows env. I want to make sure it doesn't break it for you. Could you please check? Thanks

**Comment by flipchart** [comment]
@rxaviers I'm not the OP, but can confirm this works for me on WIndows (and is required in order to get webpack building with Globalize).

**Comment by rxaviers** [comment]
Thank you @flipchart, I want to make sure it doesn't break other envs (which I guess it won't). I plan to merge this PR soon. Just finding time to do this final review.

**Comment by alunny** [comment]
Thanks @rxaviers, will confirm today.

File: util.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ var path = require("path");
 var mainFiles = ["ca-gregorian", "currencies", "dateFields", "numbers", "timeZoneNames", "units"];
 
 var isGlobalizeModule = function(filepath) {
-  filepath = filepath.split( "/" );
+  filepath = filepath.split( /[\/\\]/ );
```
Comment (alunny) [inline util.js]: You can use `path.sep` in node to get the platform specific file separator (a bit of a pain with some of the regex literals in this diff though).

**Comment by alunny** [comment]
Diff works with our app, thanks for the PR @aruberto :)

**Comment by texttechne** [comment]
I can also confirm that the fix works.

@rxaviers Any chance to get this merged and released in the near future? IMHO it is more or less a critical bug...

**Comment by rxaviers** [comment]
Thank you @aruberto and everyone that checked this fix.

**Comment by rxaviers** [comment]
Released as `0.3.2`. Sorry for the delay.

**Comment by texttechne** [comment]
wow, that was amazingly fast! Thanks!!!

**Pull request closed without merging**